ml/backend/ggml/ggml.go

```diff
@@ -297,12 +299,16 @@ func New(r *os.File, params ml.BackendParams) (ml.Backend, error) {
 		}
 	}
 
-	// concurrently read in tensor data. uses a section reader which is safe for concurrent reads
-	sr := io.NewSectionReader(r, int64(meta.Tensors().Offset), n-int64(meta.Tensors().Offset))
-	var g errgroup.Group
+	var doneBytes atomic.Uint64
+	totalBytes := uint64(n) - meta.Tensors().Offset
+
+	g, _ := errgroup.WithContext(ctx)
```

Comment:
Does this actually cancel the group members based on the passed ctx? It looks like it is the reverse - just canceling the derived context if one of the calls errors.